### ISSUE TITLE
Faker.fake("{{random.uuid}}") throws error in version 3.1 

### ISSUE THREAD

**Issue by donpark** [Marak/faker.js#368]
`Faker.fake("{{random.uuid}}")` throws:

```
TypeError: self.number is not a function
    at replacePlaceholders .../node_modules/faker/lib/random.js:93:29)
```

but not in version 3.0.1.

Calling `Faker.random.uuid()` directly also works.

String format version may be mishandling `this` binding in version 3.1.

**Comment by skylarstein** [comment]
Fixed in PR https://github.com/Marak/faker.js/pull/374

**Comment by Marak** [comment]
Should be resolved with #377 

Published in latest version of faker
